<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraAdvancedParamsWidget</class>
 <widget class="QWidget" name="CameraAdvancedParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QnPanel" name="leftPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="groupsWidget">
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QnProgressWidget" name="progressWidget" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="contentsWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/progress_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnPanel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
